integration-tests/test_file_workflow.py

```diff
@@ -46,12 +49,19 @@ def test_file_workflow_with_an_archive_with_only_one_canonical_fact(
     # once archive is uploaded system status changes to register
     assert conftest.loop_until(lambda: insights_client.is_registered)
 
-    insights_client.run("--check-results")  # to get host details from inventory
-    with open(HOST_DETAILS, "r") as data_file:
-        file_content = json.load(data_file)
-        host_data = file_content["results"][0]
-    assert host_data["insights_id"] == machine_id
-    assert host_data["fqdn"] is None
+    host_data = {}
```

Comment:
No need to combine these changes here, we have already implemented API get/post in pytest_client_tools, so if needed refactoring we will do it in a separate PR for all the --check-results occurrences.

Reply:
Yes, I will refactor it in a separate PR